package.json

```
@@ -39,6 +39,7 @@
     "lint": "npm-run-all --continue-on-error --aggregate-output --parallel lint:*",
     "lint:docs": "markdownlint \"**/*.md\"",
     "lint:eslint-docs": "npm-run-all \"update:eslint-docs -- --check\"",
+    "lint:fix": "yarn lint:js --fix && yarn lint:package-json-sorting:fix",
```

```suggestion
    "lint:fix": "npm-run-all \"lint:*:fix\"",
```

https://github.com/ember-cli/ember-cli/pull/10022/files#diff-e4c0828d65b51abb30d80befe6fe7870dc36a73dad0083a9dcbb09d48f58e1cf

---

you want to use npm-run-all instead of concurrently?

---

Yes, npm-run-all is fine. No problem with is so no urgency to switch.

---

And you'll have to add individual fix scripts for each type of linting of course.

---

done